Library/Homebrew/cmd/update.sh

```diff
@@ -2,8 +2,11 @@
 #:
 #:  Fetch the newest version of Homebrew and all formulae from GitHub using `git`(1) and perform any necessary migrations.
 #:
-#:       --merge                         `git merge` is used to include updates (rather than `git rebase`).
-#:       --force                         Always do a slower, full update check (even if unnecessary).
+#:          --merge                      `git merge` is used to include updates (rather than `git rebase`).
```

Comment:
Inline this with the `-f`.

Reply:
Really? This is an example of what I am trying to match:

```
$ brew help info | tail -n4
        --installed                  Get information on all installed formulae.
    -v, --verbose                    See more verbose analytics data.
    -d, --debug                      Display any debugging information.
    -h, --help                       Show this message.
```

Reply:
Ok, 🆒, I get this now from below!